### DISCUSSION TITLE
Referral ideas for friends / streamers

### DISCUSSION THREAD

**Discussion by ar00n** [CraftRealm/management#24] [Ideas]
Comment any suggestions for rewards / incentives for those inviting friends. For example having a code to give to viewers or friends to enter when joining.

**Comment by ar00n** [comment]
> i mean the obvious answer is diamonds or money for rewards for asking out friends to join and play
or viewers

I mean yeah but I want something that will only be given to real players, not just those that join and leave. Also don't want it to provide a substantial in game advantage as that would be unfair.

**Reply by AngelInTheHeaven** [reply]
You mean by daily rewards? Or hours marking. Where there specific amount of hours you need played not afked to get awards?
The reward can be joke weapons? Like stick with knockbackn1 or smth along those lines

**Reply by ar00n** [reply]
A minimum amount of playtime before you can claim it yes. And the person claiming it gets a reward and also the person who gave out the code gets a reward. Idk what the reward should be, but it should be something cool / fun / funny /useful but not unfair to others.

**Comment by AngelInTheHeaven** [comment]
Toilet paper  renamed 48 paper
China made diamonds renamed 1 lapis
/Commands? Like /pet /head 
Roblox new tos renamed paper (and flint and steel)
Minecraft renamed grass block
Etc etc
I'm going to sleep

**Comment by AngelInTheHeaven** [comment]
Oh forgot u mentioned useful
Sharpness 999 diamond sword with one dur and 999 lvl repair or upgrade

**Comment by AngelInTheHeaven** [comment]
Forgot to also add 

It's bloody raw renamed 64 salmon (Gordon Ramsay)

**Comment by ar00n** [comment]
Bonus percentage of sales / some profits of refer user should go towards the referrer. And vice versa possibly

**Reply by AngelInTheHeaven** [reply]
5% each refer? 25% max?

**Reply by ar00n** [reply]
Well there wouldn't be a max, nothing is taken from either person. Just that percent is given to them, idk I'll have to think about this one.

**Reply by ar00n** [reply]
> Wait what so if someone raids the server one of the people can have like 200% more money gain on sales?

@AngelInTheHeaven noooooo... They wouldn't get that bonus, they'd get a little money bonus from the person who used the code every time they sold something. So they'd both get some money off eachother when they sold stuff.

**Reply by AngelInTheHeaven** [reply]
Oh so communism
